main.py

```diff
@@ -115,6 +115,15 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: unicode. A raw string representing a route.
+        handler: BaseHandler. A callable to handle the route.
+        defaults: NoneType/dict. Default or keywords to be passed into a
```

Comment:
You can drop the `NoneType` here and mention that it's an optional argument instead.

Reply:
I dropped it.